code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -110,7 +110,7 @@
 	return
 
 /obj/item/weapon/gun/projectile/automatic/mini_uzi
-	name = "uzi"
+	name = "\improper 'Type U3' SMG"

```

Comment:
Is there some particular reason we can't call this an uzi? Uzi is the kind of name that even gun illiterate people understand